src/index.html

```diff
@@ -11,7 +11,219 @@
+            autocomplete="off"
+            id="surname"
+          >
+
+        </label>
+
+        <label for="name" class="form-field">
+            Name:
+
+          <input
+            type="text"
+            name="firstname"
+            id="name"
+            autocomplete="off"
+          >
+
+        </label>
+
+        <label class="form-field">
+          How old are You?
+
+          <input
+            type="number"
+            min="1"
+            max="100"
+            name="age"
+            id="age"
+            value="12"
+          >
+
+        </label>
+
+        <label for="date" class="form-field">
+          Full date of birth:
+
+          <input
+            type="date"
+            id="date"
+            name="birth_date"
+          >
+
+        </label>
+
+        <label for="terms" class="form-field">
+          I accept the term of the agreement
+
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms"
+            required
+          >
+
+        </label>
+
+      </fieldset>
+
+      <fieldset class="field-group">
+
+        <legend>Registration</legend>
+
+        <label for="email" class="form-field">
+          E-mail:
+
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="email"
+            id="email"
+            required
+          >
+
+        </label>
+
+        <label for="password" class="form-field">
+          Password:
+
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+            id="password"
+            required
+          >
+
+        </label>
+
+      </fieldset>
+
+      <fieldset class="field-group">
+
+        <legend>Any interesting fact about you?</legend>
+
+        <label for="cats" class="form-field">
+          Do you love cats?
+
+          <input
+            type="radio"
+            name="cats"
+            value="yes"
+          >
+            Yes
+
+          <input
+            type="radio"
+            name="cats"
+            value="no"
+          >
+            No
+
+        </label>
+
+        <label for="color" class="form-field">
+          What is your favorite color?
+
+          <input
+            type="color"
+            name="favorite_color"
+            id="color"
+          >
+
+        </label>
+
+        <label for="time" class="form-field">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            name="sleep_time"
+            id="time"
+          >
+
+        </label>
+
+        <label for="brand" class="form-field">
+          What are your favorite brand of cars?
+
+          <select
+            name="favorite_car"
+            multiple
+            required
+            id="brand">
+
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
```

Comment:
better pass lower-case values - will save you from letter-case-driven issues when processing data on server
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```